x-pack/plugins/observability/public/pages/landing/index.tsx

```diff
@@ -45,92 +40,96 @@ export function LandingPage() {
   const { core, ObservabilityPageTemplate } = usePluginContext();
   const theme = useContext(ThemeContext);
```

Comment:
We have a hook that does this: https://github.com/elastic/kibana/blob/0cb59c387ea85c36d3cd38ee84519cd0df816182/x-pack/plugins/observability/public/hooks/use_theme.tsx#L16

Reply:
I appreciate the feedback @smith, but that addition wasn't part of this PR. I'd like to keep reviews to the actual new Add Data screens if possible.